Source/Scene/Cesium3DTileset.js

```diff
@@ -63,6 +63,7 @@ define([
         var baseUrl = appendForwardSlash(url);
 
         this._url = url;
+        this._baseUrl = baseUrl;
```

Comment:
The instanced 3d tile uses it when it gets the url of an external gltf file.

https://github.com/lilleyse/cesium/blob/instancing-models-3d-tiles/Source/Scene/Instanced3DModel3DTileContentProvider.js#L159